sound/shairport-sync/Makefile

```diff
@@ -78,7 +78,8 @@ endef
 CONFIGURE_ARGS += \
 	--with-alsa \
 	--with-libdaemon \
-	--with-metadata
+	--with-metadata \
+	--with-pipe
```

Comment:
What is the size difference? Why do you need to have enabled by default? So many questions for this simple change, but no one knows why do you need to enable it. :) You know it, so please do it also for others.

Please see: https://openwrt.org/submitting-patches#submission_guidelines

Reply:
/etc/config/shairport-sync has "pipe" config support. 

Yet the shairport-sync binary is not compiled with "pipe" support by default.

I will let you know about the size  increase.

Reply:
@BKPepe 

shairport-sync-openssl_4.3.2-0_aarch64_generic.ipk without --pipe: 110806  bytes
shairport-sync-openssl_4.3.2-0_aarch64_generic.ipk with      --pipe: 111552 bytes
diff: +746 bytes

PS: Size reported with `ls -ls --block-size 1 /tmp/shairport-sync-openssl_4.3.2-0_aarch64_generic.ipk`